rest/networkProtocols/TheThingsNetwork.js

```diff
@@ -54,7 +54,7 @@ module.exports = {
           type: 'string',
           label: 'Username',
           value: '',
-          required: true,
+          required: false,
           placeholder: 'myLoraUsername',
```

Comment:
This is my fault but this should be myTTNUsername  and myTTNPassword I believe.  Otherwise looks good.